www/src/pages/de/t3-collection.md

```diff
@@ -1,15 +1,15 @@
 ---
```

Comment:
How should we handle the T3 Collection across multiple languages? 

One across all? Or language specific (companies aswell as projects)?

I prefer one collection across all